aday2dream/src/main/kotlin/com/aday2dream/aday2dream/repository/AccountRepository.kt

```diff
@@ -8,4 +8,7 @@ import org.springframework.stereotype.Repository
 @Repository
 @EnableJpaRepositories
 interface AccountRepository : JpaRepository<Account, Long> {
+    fun findByUsername(username: String) : Account?
+    fun existsByUsername(username: String) : Boolean
```

Comment:
you dont need this, you can just reuse findByUsername, if it is empty -> it does not exists